src/style.css

```diff
@@ -0,0 +1,59 @@
+body {
+  background-color: #d3d3d3;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.box-list {
+  background-color: #fff;
+  width: 50%;
+  box-shadow: 0 0.5rem 1rem rgba(0, 0, 0, 0.2);
+}
+
+.list-title {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid #d3d3d3;
+  align-items: center;
+  padding: 0 1rem;
+}
+
+.list-title h3 {
+  font-size: 1.5rem;
+}
```

Comment:
- [ ] Nice work you did on the project 👏 . However, there seems to be a little disparity between your design and that given the project requirements. Kindly find the difference in the structure below, and effect the changes appropriately 👍 

## Your design :
![image](https://user-images.githubusercontent.com/50990925/220487800-612a0391-01f4-42c2-af25-7b8bee76bba8.png)


## Figma:
![image](https://user-images.githubusercontent.com/50990925/220487768-b73457c5-c03c-4576-818b-2178351f7364.png)

Reply:
Hey Shedrack, I really don't understand what you mean. Since when I run npm start it displays the suggested templated above